### DISCUSSION TITLE
[FR] Annotate on frame range instead of 1 frame.

### DISCUSSION THREAD

**Discussion by NicTanghe** [AcademySoftwareFoundation/OpenRV#69] [Ideas]
I`d like to request a feature to allow you to select a range of frames when adding an annotation.

**Comment by savoiei** [comment]
Hi NickTanghe, for the time being, there is no option coming with OpenRV to annotate on multiple frames at once. There might be additional options that could be used if you go through the rvio tool (such as adding watermarks, frame burn-ins, adding an overlay).
